### PR TITLE
Added member profile

### LINKED ISSUE #2772: [Community] Member Profile: Maximiliano Churichi
**Current Behavior**

@mchurichi has been a consistent contributor and community member.

**Desired Situation**
Let's recognize <@github-username> as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: mchurichi
- Twitter: mchurichi
- LinkedIn:  maximilianochurichi
- Link to profile picture: https://media-exp1.licdn.com/dms/image/C4D03AQE6jtCF9k204g/profile-displayphoto-shrink_800_800/0/1516954557825?e=1655942400&v=beta&t=2OawpONFkyZnXgRo_TVyeJCxzDzMtYCJObJsXhEN_QQ

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by harshitasao**
Signed-off-by: harshitasao <harshitasao@gmail.com>

**Description**
Added profile of Maximiliano Churichi 

This PR fixes #2772 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/maximiliano-churichi/index.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+name: Maximiliano Churichi
+position: Contributor
+image_path: ./Maximiliano-Churichi.jpg
+github: mchurichi
+twitter: mchurichi
+linkedin: maximilianochurichi
+location: Paraná, Argentina
+bio: Maxi is a Sofware Engineer who loves to understand the whole picture, from the pixels on the screen to the bits that travel through the wires. Passionate about mentoring young engineers, open-source, cyber security, cloud infrastructure, and solving complex problems with simple solutions (service meshes being one of them!).
+ In his free time, Maxi loves to travel abroad as much as time permits, and spend time with his dog Beto and his cat Lola.
+badges:
```
Comment (warunicorn19) [inline src/collections/members/maximiliano-churichi/index.mdx:11]:
```suggestion
```

**Review by warunicorn19 — COMMENTED** [review]
We can just leave out this field, since it's empty.

**Comment by l5io** [comment]
🚀 Preview for commit 1e54a5802822fe68419ee4baca906f2e9ffc4d17 at: https://6280d47af8e00e239f3aead8--layer5ng.netlify.app

**Comment by warunicorn19** [comment]
@mchurichi could you please take a look here, if everything seems fine let's merge.

**Comment by mchurichi** [comment]
Looks great, thanks @harshitasao!

**Review by Chadha93 — APPROVED** [review]
Lgtm @harshitasao good work 👏

**Comment by leecalcote** [comment]
Yay! 🎈@mchurchi 🥳

**Pull request merged**